dom0/runner.py

```diff
@@ -160,6 +160,10 @@ def prepare(self):
         # synchronize dom0 clock
         self.run_cmd("sudo qvm-sync-clock")
 
+        # Install testing dependencies
+        self.run_cmd("command -v pytest || sudo qubes-dom0-update -y python3-pytest")
```

Comment:
I don't think this is being run in a shell so `||` won't work. We're already running in dom0, so we can use `if os.path.exists()`. Or maybe we just unconditionally do this, it should be relatively quick anyways.